docs/pages/templates/typescript/threejs.mdx

```diff
@@ -156,7 +160,7 @@ Both `useComponentValue` and `useEntityQuery` cause the React component in which
       <ambientLight />
       {/* eslint-disable-next-line react/no-unknown-property */}
       <pointLight position={[10, 10, 10]} />
-      <Plane position={[0, -5, 0]} />
+      <Plane />
```

Comment:
mind updating this one with `CollapseCode` while you're here?

Reply:
Done.